packages/web-app-files/tests/unit/components/SideBar/PrivateLinkItem.spec.ts

```diff
@@ -5,6 +5,16 @@ import { createStore, defaultPlugins, mount, defaultStoreMockOptions } from 'web
 
 jest.useFakeTimers()
 
+const folder = mockDeep<Resource>({
```

Comment:
We are getting rid of mockDeep where not strictly neccessary, it has weird side effects and can trigger stack overflows with vue 3

Reply:
Otherwise good if you fix the tests 💫

Reply:
We'll have to pass some sort of resource to mount the component - I don't think this test in particular will be especially picky about how the resource gets built (I've just extracted what was being used on the `getWrapper` function further down in the file 🤓 ), so should be fine when the refactoring time comes ;)

Reply:
Just use mock instead of mockDeep

Reply:
Done